### DISCUSSION TITLE
Ergo Hashrate Decreasing.

### DISCUSSION THREAD

**Discussion by lazlomo** [NebuTech/NBMiner#790] [Q&A]
on my bios moded RX570 the ergo hashrate is decreasing every month.
I used to get 65-67MHS after a month or so it suddenly dropped to 62MHS, and never reach higher hash rates.
3days ago it dropped again to 57-58.
I dunno what's happening I'm using the same OC presets I made, thermals are so much as the same 2-4 ups and downs on night and day.
can anyone help me out?

**Comment by Tom5051** [comment]
Is the difficulty level of ergo increasing?

**Reply by lazlomo** [reply]
What does that supposed to do with hashrages?

Difficulty increase will result less rewards for the same hashrates.

Hashrate is native to gpu and miner application.

Pool/algorithm/difficulty does not effect that.
